src/sweeps/bayes_search.py

```diff
@@ -222,6 +215,7 @@ def next_sample(
         predicted_y: predicted value
         predicted_std: stddev of predicted value
         expected_improvement: expected improvement
+        warnings <List[str]>: warnings encountered
```

Comment:
I think `warnings` as you have it elsewhere in the code is just a string right now, should we update this or update warnings to be list of string?